config/build_lcrc_improv.sh

```diff
@@ -0,0 +1,61 @@
+#!/bin/bash
+
+################################################################
+## * This script builds available configurations of QMCPACK   ##
+##   on Improv, at Argonne National Lab.                      ##
+##                                                            ##
+## * Execute this script in trunk/                            ##
+##   ./config/build_lcrc_improv.sh                            ##
+##                                                            ##
+## Last verified: May 15, 2024                                ##
+################################################################
+
+# module files resulting from module imports below:
+# Currently Loaded Modulefiles:
+#  1) xalt/3.0.1     (S)   4) gcc/13.2.0                 7) amdblis/4.2-gcc-13.2.0
+#  2) cmake/3.27.4         5) openmpi/5.0.2-gcc-13.2.0   8) aocl-utils/4.2-gcc-13.2.0
+#  3) .binutils/2.41 (H)   6) python/3.11.6-gcc-13.2.0   9) amdlibflame/4.2-gcc-13.2.0
+
+
+source $MODULESHOME/init/bash
+module purge
+module load cmake/3.27.4
+module load gcc/13.2.0
+module load openmpi/5.0.2-gcc-13.2.0
+module load amdlibflame/4.2-gcc-13.2.0
+module list
+
+
+CMAKE_FLAGS="-DENABLE_PPCONVERT=0 \
+             -DLAPACK_LIBRARIES=/gpfs/fs1/soft/improv/software/spack-built/linux-rhel8-zen3/gcc-13.2.0/openblas-0.3.26-blohgyt/lib/libopenblas.so \
```

Comment:
Loaded libflame but forcing openblas as the LAPACK?
Technically, amdblis is the BLAS and amdlibflame seems to be the LAPACK.
If you insist using openblas. Could you unload blis and flame?

Reply:
Changed